impl/client/src/doc/javascript/template/vizApi/tmpl/details.tmpl

```diff
@@ -54,7 +54,7 @@ if (data.defaultvalue && (data.defaultvaluetype === 'object' || data.defaultvalu
 </ul></dd>
 <?js } ?>
 
-<?js if (data.implements && data.implements.length) { ?>
+<?js if (data.kind !== "class" && data.implements && data.implements.length) { ?>
```

Comment:
This is to disable the details of implements when we are parsing a class/constructor doclet.

When you have a method that implements the corresponding method from an interface, it will appear here